index.html

```diff
@@ -0,0 +1,195 @@
+                  </div>
+                </article>
+                <article class="lecture">
+                   <div class="lecture-left">
+                     <i class="fab fa-airbnb fa-3x"></i>
+                   </div> 
+                   <div class="program-title">
+                     <h3>Forum</h3>
+                   </div>
+                   <div class="lecture-right">
+                     <p>listen to speakers from various countries about the messeges of sharing and opening.</p>
+                   </div>
+                 </article>
+                <article class="lecture">
+                   <div class="lecture-left">
+                     <i class="fab fa-airbnb fa-3x"></i>
+                    </div>
+                    <div class="program-title">
+                      <h3>Workshop</h3>
+                    </div>
+                    <div class="lecture-right">
+                      <p>listen to speakers from various countries about the messeges of sharing and opening.</p>
+                    </div>
+                 </article>
+                 <article class="lecture">
+                   <div class="lecture-left">
+                     <i class="fab fa-airbnb fa-3x"></i>
+                    </div>
+                    <div class="program-title">
+                      <h3>Ignite</h3>
+                    </div>
+                    <div class="lecture-right">
+                      <p>listen to speakers from various countries about the messeges of sharing and opening.</p>
+                    </div>
+                 </article>
+              </div>
+              
+            <div class="program-button">
+                <button type="button" class="button-program">Join Global Tec conference</button>
+            </div>
+            <div class="program-desktop">
+              <a href="/" class="program-desktop-a">SEE THE WHOLE PROGRAM</a>
+            </div>
+          </section>
+          <section class="speak" id="speak">
+            <div class="speak-h2">
+              <h2>Featured Speakers</h2>
+              <hr class="horizontal-hr">
+            </div>
+            <div class="speakers"></div>
+            <!-- <div class="cards-container">
+              <div class="container">
+                <div class="cards-image">
+                  <img src="images/MarkZuk.png" class="image-inside-cards2" alt="...">
+                </div>
+                <div class="cards-info">
+                  <div class="cards-body">
+                    <h3 class="cards-name">Card title</h3>
+                    <h5 class="cards-title">This is a wider card with supporting text below as a natural lead-in.</h5>
+                    <p class="cards-small"><small class="text-muted">agoLast updated 3 mins agoLast updated 3 mins agoLast updated 3 mins agoLast updated 3 mins agoLast updated 3 mins ago</small></p>
+                  </div>
+                </div>
+              </div>
+            </div> -->
```

Comment:
- [ ] remove this commented block of code.

Reply:
Alright, thank you!

Reply:
This issue is well resolved! ✔️